pipelines/visitQAEllipPlots.yaml

```diff
@@ -270,6 +270,29 @@ tasks:
         from lsst.analysis.drp.calcFunctors import CalcShapeSize
         from lsst.pipe.tasks.dataFrameActions import SubtractColumns
 
+  plot_shapeSizeResidsDiff_scatter_visit:
```

Comment:
Would `plot_shapeSizeResidDiff_scatter_visit` make more sense here?, i.e., not plural?